src/modm/platform/clock/stm32/rcc.hpp.in

```diff
@@ -315,6 +341,14 @@ public:
 		PllQ = RCC_{{ccipr1}}_CLK48{{sel48}}_1,
 		Msi = RCC_{{ccipr1}}_CLK48{{sel48}}_1 | RCC_{{ccipr1}}_CLK48{{sel48}}_0
 	};
+%% elif target.family in ["u5"]
+	enum class Clock48Source
+	{
+		Hsi48 = 0b00 << RCC_CCIPR1_ICLKSEL_Pos,
+		Pll1Q = 0b01 << RCC_CCIPR1_ICLKSEL_Pos,
+		Pll2Q = 0b10 << RCC_CCIPR1_ICLKSEL_Pos,
```

Comment:
`Pll1Q` is `0b10` for this setting according to the reference manual.
![iclksel](https://user-images.githubusercontent.com/25187160/161523229-ad60cef8-ad7b-4632-b009-2ac59f893536.png)

Reply:
Thanks!